src/persist/src/indexed/cache.rs

```diff
@@ -537,7 +543,8 @@ impl BlobCacheInner {
 /// perhaps should look at the in-memory size instead.
 #[derive(Debug)]
 struct BlobCacheCore<D> {
-    dataz: HashMap<String, Arc<D>>,
+    // Map from key -> (data, serialized size of data)
```

Comment:
super nit: "serialized size of data" happens to be what the single user of this is doing, but at this level of abstraction it's something more like a "weight" of this entry

Reply:
changed it to "size" as i felt "weight" is a bit confusing to the reader.